custom/icds_reports/ucr/reports/mobile_mpr_2a_deaths.json

```diff
@@ -0,0 +1,385 @@
+  "config": {
+    "title": "MPR 2a - Deaths by Month  (Static)",
+    "description": "Details of Deaths during the month.  Displays person cases grouped by month of death and owner ID.",
+    "visible": false,
+    "aggregation_columns": [
+      "owner_id", "month"
+    ],
+    "filters": [
+      {
+        "compare_as_string": false,
+        "datatype": "date",
+        "required": false,
+        "display": "Date of Death",
+        "field": "date_death",
+        "type": "date",
+        "slug": "date_death"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "awc_id",
+        "field": "awc_id",
+        "choice_provider": {
+          "type": "location",
+          "include_descendants": true,
+          "order_by_hierarchy": true,
+          "show_full_path": true
+        },
+        "display": "Filter by AWW"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "supervisor_id",
+        "field": "supervisor_id",
+        "choice_provider": {
+          "type": "location",
+          "include_descendants": true,
+          "order_by_hierarchy": true,
+          "show_full_path": true
+        },
+        "display": "Filter by Supervisor"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "block_id",
+        "field": "block_id",
+        "choice_provider": {
+          "type": "location",
+          "include_descendants": true,
+          "order_by_hierarchy": true,
+          "show_full_path": true
+        },
+        "display": "Filter by Block"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "district_id",
+        "field": "district_id",
+        "choice_provider": {
+          "type": "location",
+          "include_descendants": true,
+          "order_by_hierarchy": true,
+          "show_full_path": true
+        },
+        "display": "Filter by District"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "state_id",
+        "field": "state_id",
+        "choice_provider": {
+          "type": "location",
+          "include_descendants": true,
+          "order_by_hierarchy": true,
+          "show_full_path": true
+        },
+        "display": "Filter by State"
+      }
+    ],
+    "columns": [
+      {
+        "sortable": false,
+        "description": null,
```

Comment:
nit: I've been trying to keep any new data sources/reports clean of default values so that they're a little easier to understand on first pass (though that could bring up explicit vs implicit)

Reply:
"description" seems like a pretty innocent one to omit, as the default is pretty obvious.  This first commit was just a copy/paste of the old report, so I didn't think too hard about the representation here.

Reply:
https://github.com/dimagi/commcare-hq/pull/21278/commits/4bcc03849b683b13bdc6bce30820889eb6dd1e23 and https://github.com/dimagi/commcare-hq/pull/21278/commits/2b7e9d0f0038f29589da0077e5cda5f731914dcc